File: C/beginner/1074.cpp
#include <stdio.h>
 
int main() {
    int n;
    
    scanf("%d", &n);
    
    int num[n];
    
    for(int i = 0; i < n; i++) {
        scanf("%d", &num[i]);
    }
    
    for(int j = 0; j < n; j++) {
        if(num[j] == 0) {
            printf("NULL\n");
        } else {
            if(num[j] % 2 == 0) {
                printf("EVEN ");
            } else {
                printf("ODD ");
            }
            
            if(num[j] > 0) {
                printf("POSITIVE\n");
            } else {
                printf("NEGATIVE\n");
            }
        }
    }
 
    return 0;
}

File: C#/begginer/1074.cs
using System; 

class URI {

  static void Main(string[] args) { 
    int loopTimes = int.Parse(Console.ReadLine());
    string[] answers = new string[loopTimes];

    for(int i = 0; i < loopTimes; i++) {
      int input = int.Parse(Console.ReadLine());

      if(input == 0) answers[i] = "NULL";
      else {
        if(input % 2 == 0) answers[i] = "EVEN";
        else answers[i] = "ODD";

        if(input > 0) answers[i] += " POSITIVE";
        else answers[i] += " NEGATIVE";
      }
    }

    foreach(string answer in answers) {
      Console.WriteLine(answer);
    }

  }

}
